### PR TITLE
Business view

### PR DESCRIPTION
**Pull request by ZeeshaanMaudar**
Create Business Profile View - UI
 - Still requires logic to be added for 2 icons based based on Business type and Business Status.

### CODE REVIEW

File: biz_portal/apps/portal/templates/portal/business.html [hunk 1]
```diff
@@ -1,5 +1,134 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="{% static 'biz-portal.bundle.css' %}">
+    <title>Business Directory</title>
+  </head>
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: from DOCTYPE to here should be in base.html

File: biz_portal/apps/portal/templates/portal/business.html [hunk 2]
```diff
@@ -1,5 +1,134 @@
...
+  <!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="{% static 'biz-portal.bundle.css' %}">
+    <title>Business Directory</title>
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: this should be in {% block title %}

File: biz_portal/apps/portal/templates/portal/business.html [hunk 3]
```diff
@@ -1,5 +1,134 @@
...
+
+  <!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="{% static 'biz-portal.bundle.css' %}">
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: this should be in `block css` or something like that - whatever's defined in base.html

File: biz_portal/apps/portal/templates/portal/business.html [hunk 4]
```diff
@@ -1,5 +1,134 @@
...
+                  </summary>
+                  <span class="mdc-list-item">
+                    <span class="mdc-expansion__header">{{ object.physical_address }}</span>
+                  </span>
+                </details>
+                <details class="mdc-theme--background mdc-elevation--z2">
+                  <summary class="mdc-expansion__summary">
+                    <span class="business-profile-location mdc-list-item">
+                      <span class="status-icon-span">
+                        <img class="status-icon" src="../../../../static/images/ellipse.svg" />
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: this should be in `portal/static` then we can use `{% static 'portal/static/... %}`

File: biz_portal/apps/portal/templates/portal/business.html [hunk 5]
```diff
@@ -1,5 +1,134 @@
...
+                      </span>
+                      <span class="type-header">Status:</span>
+                      <span class="mdc-expansion__header">{{ object.status }}</span>
+                    </span>
+                    <div data-mdc-tooltip="This is how a tooltip will display when clicked" >
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: This is a placeholder. Please remove til we have actual text to put here

File: biz_portal/apps/portal/templates/portal/business.html [hunk 6]
```diff
@@ -1,5 +1,134 @@
...
+                      <i class="material-icons mdc-list-item__graphic" aria-hidden="true">info</i>
+                    </div>
+                  </summary>
+                </details>
+                <details class="mdc-theme--background mdc-elevation--z2">
+                  <summary class="mdc-expansion__summary">
+                    <span class="business-profile-location mdc-list-item">
+                        <span class="registration-icon-span">
+                          <img class="registration-icon" src="../../../../static/images/content-paste.svg" />
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: {% static... %}

File: biz_portal/apps/portal/templates/portal/business.html [hunk 7]
```diff
@@ -1,5 +1,134 @@
...
+                <button class="mdc-button more-details-button">
+                  <span class="mdc-button__label">Claim this business</span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </main>
+
+      <script src="{% static 'biz-portal.bundle.js' %}" defer></script>
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: should be in script block

File: biz_portal/apps/portal/templates/portal/business.html [hunk 8]
```diff
@@ -1,5 +1,134 @@
...
+  </body>
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business.html]: /body and /html should only be in base, not inside content blocik

File: biz_portal/apps/portal/templates/portal/business.html (hunk 1 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Done

File: biz_portal/apps/portal/templates/portal/business.html (hunk 2 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Please let me know if title should change for different pages. Let me know the content update.

File: biz_portal/apps/portal/templates/portal/business.html (hunk 3 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Covered

File: biz_portal/apps/portal/templates/portal/business.html (hunk 4 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Covered

File: biz_portal/apps/portal/templates/portal/business.html (hunk 5 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Covered

File: biz_portal/apps/portal/templates/portal/business.html (hunk 6 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Covered

File: biz_portal/apps/portal/templates/portal/business.html (hunk 7 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Done

File: biz_portal/apps/portal/templates/portal/business.html (hunk 8 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business.html]: Covered

File: biz_portal/templates/base.html [hunk 9]
```diff
@@ -22,13 +26,51 @@
...
+    <main class="main-content" id="main-content">
+      <div class="mdc-top-app-bar--fixed-adjust top-bar">
     {% block content %}
 
     {% endblock content %}
+      </div>
+    </main>
+
 
+    <script src="{% static 'biz-portal.bundle.js' %}" defer></script>
```
Comment (jbothma) [inline biz_portal/templates/base.html]: should be inside javascript block

**Pull request merged**